variables.tf

```diff
@@ -25,5 +25,6 @@ variable "project" {
     description = "My project"
     purpose     = "Personal"
     environment = "development"
+    is_default  = "false"
```

Comment:
This is a boolean - could you please update the map with the correct signature to include it?

Furthermore, the default project should be the default, so I would set the default of `is_default` to `true`.
Thanks!